docs/src/tutorials/conic/ellipse_approx.jl

```diff
@@ -158,10 +135,128 @@ Test.@test isapprox(c, [-3.24802, -1.842825]; atol = 1e-2)                    #s
 
 P = sqrt(D)
 q = -P * c
+data = [tuple(P \ [cos(θ) - q[1], sin(θ) - q[2]]...) for θ in 0:0.05:(2pi+0.05)]
+Plots.plot!(plot, data; c = :crimson, label = nothing)
 
-Plots.plot!(
-    plot,
-    [tuple(P \ [cos(θ) - q[1], sin(θ) - q[2]]...) for θ in 0:0.05:(2pi+0.05)];
-    c = :crimson,
-    label = nothing,
-)
+# ## Alternative formulations
+
+# The formulation of `model` uses [`MOI.RootDetConeSquare`](@ref). However,
+# because SCS does not natively support this cone, JuMP automatically
+# reformulates the problem into an equivalent problem that SCS _does_ support.
+# You can see the reformulation that JuMP chose using [`print_active_bridges`](@ref):
+
+print_active_bridges(model)
+
+# There's a lot going on here, but the first bullet is:
+# ```raw
+# * Unsupported objective: MOI.VariableIndex
+# |  bridged by:
+# |   MOIB.Objective.FunctionizeBridge{Float64}
+# |  introduces:
+# |   * Supported objective: MOI.ScalarAffineFunction{Float64}
+# ```
+# This says that SCS does not support a `MOI.VariableIndex` objective function,
+# and that JuMP used a [`MOI.Bridges.Objective.FunctionizeBridge`](@ref) to
+# convert it into a `MOI.ScalarAffineFunction{Float64}` objective function.
+
+# We can leave JuMP to do the reformulation, or we can rewrite our model to
+# have an objective function that SCS natively supports:
+
+@objective(model, Max, 1.0 * t + 0.0);
+
+# Re-printing the active bridges:
+
+print_active_bridges(model)
+
+# we get `* Supported objective: MOI.ScalarAffineFunction{Float64}`.
+
+# We can manually implement some other reformulations to change our model to
+# something that SCS more closely supports by:
+#
+#  * Replacing the [`MOI.VectorOfVariables`](@ref) in [`MOI.PositiveSemidefiniteConeTriangle`](@ref)
+#    constraint `@variable(model, Z[1:n, 1:n], PSD)` with the
+#    [`MOI.VectorAffineFunction`](@ref) in [`MOI.PositiveSemidefiniteConeTriangle`](@ref)
+#    `@constraint(model, Z >= 0, PSDCone())`.
```

Comment:
```suggestion
#    `@constraint(model, Symmetric(Z) >= 0, PSDCone())`.
```

Reply:
Alternatively you can do `@variable(model, Z[1:n, 1:n], Symmetric)` and then `@constraint(model, Z >= 0, PSDCone())`